docs/excel/custom-functions-runtime.md

```diff
@@ -72,26 +72,27 @@ ws.onerror = function (error) {
+Within a custom function (or within any other part of an add-in), you can store and access data by using the `Office.Storage` object. `Office.Storage` is a persistent, unencrypted, key-value storage system that provides an alternative to [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage), which cannot be used within custom functions. `Office.Storage` offers 10 MB of data per domain. Domains can be shared by more than one add-in.
 
-`AsyncStorage` is intended as a shared storage solution, meaning multiple parts of an add-in are able to access the same data. For example, tokens for user authentication may be stored in `AsyncStorage` because it can be accessed by both a custom function and add-in UI elements such as a task pane. Similarly, if two add-ins share the same domain (e.g. www.contoso.com/addin1, www.contoso.com/addin2), they are also permitted to share information back and forth through `AsyncStorage`. Note that add-ins which have different subdomains will have different instances of `AsyncStorage` (e.g. subdomain.contoso.com/addin1, differentsubdomain.contoso.com/addin2). 
+`Office.Storage` is intended as a shared storage solution, meaning multiple parts of an add-in are able to access the same data. For example, tokens for user authentication may be stored in `Office.Storage` because it can be accessed by both a custom function and add-in UI elements such as a task pane. Similarly, if two add-ins share the same domain (e.g. www.contoso.com/addin1, www.contoso.com/addin2), they are also permitted to share information back and forth through `Office.Storage`. Note that add-ins which have different subdomains will have different instances of `Office.Storage` (e.g. subdomain.contoso.com/addin1, differentsubdomain.contoso.com/addin2).
 
-Because `AsyncStorage` can be a shared location, it is important to realize that it is possible to override key-value pairs.
+Because `Office.Storage` can be a shared location, it is important to realize that it is possible to override key-value pairs.
 
-The following methods are available on the `AsyncStorage` object:
+The following methods are available on the `Office.Storage` object:
  
  - `getItem`
+ - `getItems`
  - `setItem`
+ - `setItems`
  - `removeItem`
- - `getAllKeys`
- - `flushGetRequests`
- - `multiGet`
- - `multiSet`
- - `multiRemove`: You will note that there is no implementation of a method for clearing all information (such as `clear`). Instead, you should instead use `multiRemove` to remove multiple entries at a time.
+ - `removeItems`
+ - `getKeys`
 
-### AsyncStorage example 
+You will note that there is no implementation of a method for clearing all information (such as `clear`). Instead, you should instead use `removeItems` to remove multiple entries at a time.
```

Comment:
Consider changing this to a !TIP or something. You could also be more direct be saying something like "To remove all the information, use `removeItems` on every entry."

Reply:
There is similar API that some devs are used to using that includes `clear` and given our structure is so similar to that other API it's worth calling out